src/Apps/ArtAdvisor/07-Curated-Discovery/Components/Result/Result.tsx

```diff
@@ -0,0 +1,18 @@
+import { Box } from "@artsy/palette"
+import { Action, State } from "Apps/ArtAdvisor/07-Curated-Discovery/App"
+import { FC } from "react"
+
+interface ResultProps {
+  state: State
+  dispatch: React.Dispatch<Action>
+}
+
+export const Result: FC<ResultProps> = props => {
+  const { state } = props
+
+  return (
+    <Box>
+      <pre>{JSON.stringify(state, null, 2)}</pre>
+    </Box>
+  )
+}
```

Comment:
Placeholder just blasts the state to the screen for now, this is where we'll drop in some rails that read from the global state on mount, and fetch the appropriate results.